pr_agent/tools/pr_description.py

```diff
@@ -0,0 +1,83 @@
+import copy
+import json
+import logging
+
+from jinja2 import Environment, StrictUndefined
+
+from pr_agent.algo.ai_handler import AiHandler
+from pr_agent.algo.pr_processing import get_pr_diff
+from pr_agent.algo.token_handler import TokenHandler
+from pr_agent.algo.utils import convert_to_markdown
+from pr_agent.config_loader import settings
+from pr_agent.git_providers import get_git_provider
+from pr_agent.git_providers.git_provider import get_main_pr_language
+
+
+class PRDescription:
```

Comment:
Consider adding more comments in the code to explain the logic behind the PRDescription class and its methods. This will make the code more readable and maintainable. [important]